tests/openvino/native/quantization/test_fq_configurations.py

```diff
@@ -1,18 +0,0 @@
-# Copyright (c) 2023 Intel Corporation
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#      http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-from nncf.openvino.statistics.statistics import OVMinMaxTensorStatistic
-from tests.post_training.test_templates.test_calculate_quantizer_parameters import TemplateTestFQParams
-
-
-class TestFQParams(TemplateTestFQParams):
-    @property
-    def tensor_statistic(self):
-        return OVMinMaxTensorStatistic
```

Comment:
Patterns such as this are no longer required to test the common code. In fact, the common code could be tested in the `tests/common` scope, by instantiating and running the `TemplateTestFQParams` as an actual test class there, powered by the `NPNNCFTensor`, and the backend-specific tests in `tests/openvino`, `tests/torch` etc. should instead be written to concentrate on running the backend-specific code for testing.